steps/apply.ts

```diff
@@ -35,3 +35,15 @@ export const completeFundingInformationTask = async (page: Page, name: string) =
   await fundingInformationPage.checkRadio('Personal money or savings')
   await fundingInformationPage.clickSave()
 }
+
+export const completeAboutThePersonTask = async (page: Page, name: string) => {
```

Comment:
query: should this be `completeEqualityAndDiversityTask`? The 'About the Person` is the section?

Reply:
good point! thank you

I've had another go at this and am now proposing "per-section" steps at the top level in our "Create application" feature. I've extracted the task-level step helpers to section-specific files. Hopefully this is pattern which will be clear to maintain and build on?